safety_timeout: 5.0 # [s]
main_timer_rate: 30 # [Hz]

goto:
  x: 0.0
  y: 0.0
  z: 0.0
  yaw: 1.57

# delay between actions
action_delay: 3.0 # [s]
